composable.yml

```diff
@@ -8,6 +8,14 @@
       shell: |
         cp   {{ oc_deploy_template }} {{ infrared_oc_deploy_template }}
 
+    - name: override the overcloud_ceph_deploy.sh.j2 for composable roles
+      vars:
+        oc_ceph_deploy_template: "{{ ansible_user_dir }}/jetpack/templates/composable_roles_overcloud_ceph_deploy.sh.j2"
+        infrared_oc_ceph_deploy_template:  "{{ ansible_user_dir }}/.infrared/plugins/tripleo-overcloud/templates/overcloud_ceph_deploy.sh.j2"
+      shell: |
+        cp   {{ oc_ceph_deploy_template }} {{ infrared_oc_ceph_deploy_template }}
+      when: osp_release|int >= 17
```

Comment:
add check as this task has to be executed only when ceph enabled

Reply:
added the changes